Gemfile

```diff
@@ -79,6 +79,10 @@ gem 'daemons'
 
 gem 'mongoid_rails_migrations', git: 'https://github.com/adacosta/mongoid_rails_migrations.git', branch: 'master'
 
+# Cypress doesn't directly depend on nokogiri, but several of Cypress's dependencies do.
+# This is here to force version 1.6.8, because bundler was resolving it to 1.6.8.1, which has security vulnerabilities
+gem 'nokogiri', '1.6.8'
```

Comment:
@okeefm I have a PR open to HDS `master` because @ssayer requested it (to then be cherry-pick'd on `mongoid5`). Currently waiting to hear about integration issues with Bonnie. If it doesn't work out or if they need more time and we want to unblock our PRs, we could just make the change in `mongoid5`. Let me know if you think this is what we should do.